lib/evaluate.js

```diff
@@ -62,12 +62,14 @@ export const evaluate = async ({
     requiredCommitteeSize,
     logger
   })
-  const honestMeasurements = measurements.filter(m => m.fraudAssessment === 'OK')
+  const measurementsToReward = measurements.filter(
+    m => m.taskingEvaluation === 'OK' && m.majorityEvaluation === 'OK'
+  )
```

Comment:
TODO: add a test to verify that we don't reward measurements that are not in majority.

Reply:
Added in 11d896e.

Test failure when I comment out the majority-evaluation condition:

```
 1) evaluate
       rewards majority measurements only:

      AssertionError [ERR_ASSERTION]: Expected values to be strictly deep-equal:
+ actual - expected

  [
+   '0x000000000000000000000000000000000000dEaD',
    '0x10',
    '0x20',
+   '0x30'
  ]

      + expected - actual

       [
      -  "0x000000000000000000000000000000000000dEaD"
         "0x10"
         "0x20"
      -  "0x30"
       ]
      
      at Context.<anonymous> (file:///Users/bajtos/src/spark/evaluate/test/evaluate.js:277:12)
      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
```